xbmc/dialogs/GUIDialogSimpleMenu.cpp

```diff
@@ -90,12 +91,31 @@ bool CGUIDialogSimpleMenu::ShowPlaySelection(CFileItem& item, const std::string&
 
   CFileItemList items;
 
-  if (!GetDirectoryItems(directory, items, XFILE::CDirectory::CHints()))
+  if (item.GetVideoContentType() == VideoDbContentType::EPISODES)
   {
-    CLog::Log(LOGERROR,
-              "CGUIWindowVideoBase::ShowPlaySelection - Failed to get play directory for {}",
-              directory);
-    return true;
+    // See if can simplify titles
+    // Get episodes on disc
+    std::vector<CVideoInfoTag> episodes;
+    CVideoDatabase database;
+    database.Open();
+    database.GetEpisodesByFile(item.GetPath(), episodes);
+    database.Close();
+
+    const CURL url(directory);
+    auto br = XFILE::CBlurayDirectory();
+    br.GetEpisodeDirectory(url, item, items, episodes);
```

Comment:
I don't think you need an extra var for the directory as you do not use the value elsewhere.